confobo/models/event.py

```diff
@@ -1,7 +1,37 @@
+from icalendar.cal import Event as ICalEvent
+
+from confobo.config import DATETIME_FORMAT
+
+
 class Event:
-    def __init__(self, id, description=None):
-        self._id = id
+    def __init__(self, *args, **kwargs):
+
+        id_ = description = dt_start = dt_end = location = None
+
+        if args:
+            if len(args) > 1 or type(args[0]) is not ICalEvent:
+                raise TypeError()
+            ical_event = args[0]
+            id_ = ical_event.get('uid')
+            description = ical_event.get('summary')
+            dt_start = ical_event.get('dtstart').dt
+            dt_end = ical_event.get('dtend').dt
+            location = ical_event.get('location')
```

Comment:
Again, I can't foresee the best way to write `Event.__init__` since we don't actually use this anywhere yet. Just did what seemed natural at the time.